src/resourceName/routes/resourceNameRouter.ts

```diff
@@ -12,4 +12,6 @@ const resourceNameRouterFactory: FactoryFunction<Router> = (dependencyContainer)
   return router;
 };
 
+export const resourceNameRouterSymbol = Symbol('resourceNameRouterFactory');
```

Comment:
I think it should be capped as it is a const

Reply:
capped